src/gwt/src/org/rstudio/studio/client/common/shell/ShellWidget.java

```diff
@@ -499,12 +532,21 @@ public boolean execute()
                      break;
                }
                if (!canContinue)
-                  return false;
+               {
+                  break;
+               }                  
             }
-            if (!DomUtils.selectionExists())
-               scrollPanel_.scrollToBottom();
```

Comment:
When I profiled the console replay performance a while back, this turned out to be one of the biggest contributors to the replay time, so I am delighted to see it gone! Now that we play everything back in order, does the scroll panel keep up automatically (or is some other mechanism taking care of scrolling to the bottom once replay is done?)